src/client/automation/playback/move/index.js

```diff
@@ -12,6 +12,7 @@ import nextTick from '../../utils/next-tick';
 import AutomationSettings from '../../settings';
 import DragAndDropState from '../drag/drag-and-drop-state';
 import createEventSequence from './event-sequence/create-event-sequence';
+import lastHoveredElement from '../last-hovered-element';
```

Comment:
Maybe `hoveredElementTracker` is a better name for this object.

Reply:
It's not a tracker. The role of the `lastHoveredElement` is a store of the last hovered element.
Previously, the element was stored in the MoveAutomation. At present, it's necessary to have access to last hovered element from several modules.
If you don't like `lastHoveredElement` name, then I can propose these variants: `lastHoveredElementStore`, `lastHoveredElementHolder`.

Reply:
lastHoveredElementHolder is fine IMO.

Reply:
Ok. Let it be.